turtlebot/891cb68/891cb68.bug

```diff
@@ -62,4 +62,4 @@ fault-codes:
   - BDO:DEPENDENCY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
@ipa-hsd @ChrisTimperley 
Similar to the issues below. It might drop user commands, but also drops all other commands from scripts, etc.
Maybe loss of functionality is more appropriate, as a replacement or in addition to Unresponsive.

Reply:
Let's go with both loss of functionality and unresponsive.